pkg/manager/impl/validation/project_validator.go

```diff
@@ -24,6 +24,12 @@ func ValidateProjectRegisterRequest(request admin.ProjectRegisterRequest) error
 	if err := ValidateEmptyStringField(request.Project.Id, projectID); err != nil {
 		return err
 	}
+	// Note: a bit confusingly named; the project that is registered should be the same as a project update
```

Comment:
can you just call ValidateProjectUpdateRequestLabelsAlphanumeric directly on the labels so we can avoid the confusing name? (and maybe call it ValidateLabelsAlphanumeric or something not referencing Update too :-) )

Reply:
Yeah, that seems like a good idea. Let me get through wiring up unit/integration tests with the current revision and then I'll modify the signatures.

Reply:
sg, feel free to ping when you're ready for a review